pom.xml

```diff
@@ -1569,6 +1570,12 @@
         <artifactId>aircompressor</artifactId>
         <version>${aircompressor.version}</version>
       </dependency>
+
+      <dependency>
+        <groupId>com.github.davidmoten</groupId>
+        <artifactId>geo</artifactId>
+        <version>${geo.version}</version>
+      </dependency>
```

Comment:
Can we implement the same without relaying on this third party library? This is a one man project almost nobody use (see [mvnrepository](https://mvnrepository.com/artifact/com.github.davidmoten/geo/0.8.0/usages)).

Reply:
See https://yatmanwong.medium.com/geohash-implementation-explained-2ed9627a61ff

Reply:
In case we prefer to use a dependency, I would recommend to use [elasticsearch-geo](https://mvnrepository.com/artifact/org.elasticsearch/elasticsearch-geo), which at least is supported by a large company.

Specifically, they have a [GeoHash.stringEncode](https://artifacts.elastic.co/javadoc/org/elasticsearch/elasticsearch-geo/7.3.0/org/elasticsearch/geo/utils/Geohash.html#stringEncode(double,double)) function. Anyway, I would prefer to have our own implementation.

Reply:
Hmmm, yeah makes sense. I saw that though the lib was updated in apr 2024 so still being maintained
but your point is valid. I'll do that

Reply:
+1 to not taking a library dependency, especially if the impl is standalone (which seemed to be the case).

Reply:
Cool. I have incorporated these changes now